app/src/test/kotlin/com/thoughtbot/tropos/testUtils/MockCondition.kt

```diff
@@ -0,0 +1,33 @@
+package com.thoughtbot.tropos.testUtils
+
+import android.location.Location
+import com.thoughtbot.tropos.data.Condition
+import com.thoughtbot.tropos.data.Icon
+import com.thoughtbot.tropos.data.Unit
+import com.thoughtbot.tropos.data.WindDirection
+import java.util.*
+
+
+class MockCondition {
```

Comment:
I realize in the tests i called the variables `mockCondition` but we aren't mocking here, rather we are faking so maybe change to `Fake`? Or we maybe we should wait to do that all at once in a later PR?

I'm also wondering if there is a way to create package private extension functions - which would eliminate the need for the `companion object`. And we could just have `Condition.createFake()`? I'm not sure if that's a great signature though? Just thinking out loud here in word form ha

Reply:
Sure, I don't mind renaming "mock" to "fake".

`Condition.createFake()` would have to be an extension on the companion object, right? Something like:

```kotlin
fun Condition.Companion.createFake() {
  val date = TODO()
  // ...
  return Condition(...)
}
```

I haven't explored `internal` visibility yet -- I wonder if that will help us scope such an extension to only the test suite.

Reply:
Actually, what about exporting a `fakeCondition()` function from `com.thoughtbot.tropos.testUtils`? Gonna try that.